google-cloud-spanner/src/test/java/com/google/cloud/spanner/it/slow/ITBackupTest.java

```diff
@@ -467,7 +467,7 @@ public void test02_RetryNonIdempotentRpcsReturningLongRunningOperations() throws
         }
         try {
           client.cancelOperation(op.getName());
-        } catch (SpannerException e) {
+        } catch (ExecutionException e) {
```

Comment:
Are you sure that only `ExecutionException` can be thrown in this case? My hunch would be that `op.getName()` could throw `ExecutionException` (as that one needs to wait for the operation to return an initial response), while `client.cancelOperation` could throw a `SpannerException` if cancelling the operation would fail.

```suggestion
        } catch (SpannerException | ExecutionException e) {
```

Reply:
Your point makes sense to me. We can catch both exceptions here, to account for both the scenarios.